### DISCUSSION TITLE
[2132850]Rabbit and Steel 兔子与钢铁  5.17更新

### DISCUSSION THREAD

**Discussion by SamleTT** [pjy612/SteamManifestCache#663] [Share]
### Steam 游戏ID

2132850

### 密钥列表

```hash
2132851;c0cf251a04e11e7860d8bf6a88a97db33bacf450670116ca72fa7a86758b5192



```

### 清单附件

[2132851_8166276852614631453.zip](https://github.com/pjy612/SteamManifestCache/files/15344646/2132851_8166276852614631453.zip)

### 备注

code：28152105BF8B13CC
邮箱：520785@outlook.com

![image](https://github.com/pjy612/SteamManifestCache/assets/90303347/35b451dc-d7ff-4dae-b271-22f4340a9f78)

**Comment by pjy612** [comment]
* 密钥列表 数据多余
* 密钥格式不对
* 游戏ID 不匹配

**Reply by SamleTT** [reply]
修改了
